tests/Unit/Controller/Implementation/ConfigImplementationTest.php

```diff
@@ -88,6 +88,7 @@ public function testList(): void {
 			'folder' => $folder,
 			'update_interval' => $interval,
 			'print_image' => $printImage,
+			'visibleInfoBlocks' => array(),
```

Comment:
This is not sufficient to test it on PHP level. Please add a few test cases in the data provider at line 147.